File: P120956.py
def solution(babbling):
    answer = 0
    says = ["aya", "ye", "woo", "ma"]
    for cur_str in babbling:
        cut_str = cur_str

        for say_word in says:
            if say_word in cur_str:
                cut_str = cut_str.replace(say_word, " ")

        if len(cut_str.strip()) == 0 :
            answer += 1

    return answer

File: P120956.java
class P120956 {
    public int solution(String[] babbling) {
        int answer = 0;
        String[] says = new String[] {"aya", "ye", "woo", "ma"};
        for (String str : babbling) {
            String cut = str;
            
            for (String say : says) {
                if (str.contains(say)) {
                    cut = cut.replace(say, " ");
                }
            }
            
            if (cut.trim().length() == 0) answer += 1;
        }
        return answer;
    }
}